distribution/lib/Standard/Base/0.2.32-SNAPSHOT/src/Error/Problem_Behavior.enso

```diff
@@ -51,7 +51,9 @@ attach_as_needed decorated_value problem_behavior ~payload warnings=Warnings.def
 
    TODO: the Warning_System argument is temporary, as the warning system is
    mocked until the real implementation is shipped. It will be removed soon.
-attach_issues_as_needed : Any -> Problem_Behavior -> Vector -> Warning_System -> Any
-attach_issues_as_needed decorated_value problem_behavior issues warnings=Warnings.default =
-    issues.fold decorated_value value-> issue->
+attach_issues_as_needed : Problem_Behavior -> Vector -> Warning_System -> Any -> Any
+attach_issues_as_needed problem_behavior issues warnings ~decorated_value =
```

Comment:
does it make sense to make this a method of `Problem_Behavior`?

Reply:
It can definitely be done.
I'm not sure if it should be.
It looks weird for me to have `Report_Error.attach_issues_as_needed issues warnings value` -  why is `Report_Error` the receiver here?
But in practice may never have a constant there, so it will be `problem_behavior.attach_issues_as_needed issues warnings value` and I guess that looks ok.